### DISCUSSION TITLE
Create filter for `SiteTitle`

### DISCUSSION THREAD

**Discussion by blue2cat** [microsoft-search/pnp-modern-search#2848] [Q&A] [answered]
I would like to implement a filter for the `SiteTitle` property. I have linked the results webpart with the filters webpart, but the `SiteTitle` property does not show up in any searches. The `SiteURL` property appears with no issues, so I'm wondering what makes the `SiteTitle` field any different. I can see the `SiteTitle` property in the debug mode for the `PnP Search Results` webpart, so it is present. I have looked around the existing discussions, and I have not been able to locate any specific guidance that has been helpful. Any information or guidance would be welcome. I'm happy to provide more information if needed.

**Comment by kasperbolarsen** [comment, marked as answer]
Have you mapped Site Title to a RefinableString? Like in this one https://github.com/microsoft-search/pnp-modern-search/discussions/1939

**Reply by blue2cat** [reply]
I had tried the steps outlined in #1939, but I had not reindexed the site. That fixed the issue, thanks!
